upgrade/1.3.1/README.md

```diff
@@ -0,0 +1,168 @@
+# CSM 1.3.1 Patch Installation Instructions
+
+## Introduction
+
+This document guides an administrator through the patch update to Cray Systems Management `v1.3.1` from `v1.3.0`.
+If upgrading from CSM `v1.2.2` directly to `v1.3.1`, follow the procedures described in [Upgrade CSM](../README.md) instead.
+
+## Bug Fixes and Improvements
+
+* Update cfs-operator for fixed session memory limits.
+* Fix HSN NIC numbering in SMD for devices managed by HPE Proliant iLO (Redfish).
+* Restrict accessible Keycloak endpoints in OPA Policy. Require use of CMN LB for Keycloak master realm or admin.
+* Fix Prometheus error with web hook for node exporter fix.
+* Add support for collapsing session layers in CFS.
+* Mitigate security issue for kea regarding RFC 8357 support.
+* Increase CoreDNS forwarding `max_concurrent` tuning to `10000`.
+* Remove deprecated Kubernetes API use in cray-dns-unbound.
+* Allow BOS `non-rootfs_providers` to specify `root=<values>` in parameters `sessiontemplates`.
+* Mitigate Keycloak vulnerability CVE-2020-10770 via OPA Policy (API AuthZ).
+* Add a message key (hms-collector) to kafka messages to ensure events are sent to the same Kafka partition. The message key is the BMC Xname concatenated with the Redfish Event Message ID. For example `x3000c0s11b4.EventLog.1.0.PowerStatusChange`.
+* Update FAS actions test to only require at least one 'Ready' BMC.
+* Add TPM configuration support in SCSD.
+* Remove Hexane repo from RPM index.
+* Move spire jwks URL in cray-opa to ingress gateway.
+* Add Slingshot health event streaming to cray-hms-collector.
+* Fix unbound forward to PowerDNS does not working in an air-gapped configuration.
+* Update cfs-operator to remove the high priority on pods.
+* Add cray-console-* timeout to allow more time for post-upgrade hooks to complete.
+* Increase cray-dhcp-kea timeout on readiness check to from default value.
+* Fix stuck sessions during staged shutdown operations in BOS v2.
+* Add 'retry' in IMS when fetching files from S3.
+* Update documentation to run Ceph latency adjustment script during install and upgrade.
+* Add documentation to remove private key from SLS `loadstate` and `dumpstate`.
+* Add documentation and tooling for Ceph latency recovery.
+* Add documentation to resolve issue with Unbound not forwarding to PowerDNS in air-gapped configurations.
+* Add documentation to increase `fs.inotify.max_user_watches` on Kubernetes worker nodes in response to `kubectl logs -f` returning `no space` errors.
+
+## Steps
+
+1. [Preparation](#preparation)
+1. [Setup Nexus](#setup-nexus)
+1. [Upgrade services](#upgrade-services)
+1. [Verification](#verification)
+1. [Complete upgrade](#complete-upgrade)
+
+## Preparation
+
+1. Start a typescript on `ncn-m001` to capture the commands and output from this procedure.
+
+   ```bash
+   ncn-m001# script -af csm-update.$(date +%Y-%m-%d).txt
```

Comment:
All the prompts need to be moved out of the code blocks in order to be copy-paste friendly.

https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-location